modules/porous_flow/doc/content/modules/porous_flow/hysteresis.md

```diff
@@ -244,3 +244,79 @@ The result is illustrated in [histeretic_krel_extended_fig], using a cubic exten
 !media media/porous_flow/hysteretic_krel_example_1.png caption=Example hysteretic relative permeability functions.  The system initialises at full saturation, dries to saturation 0.5 (greater than $S_{l, r}$) and then wets back to saturation 1.  id=histeretic_krel_example_1
 
 !media media/porous_flow/hysteretic_krel_example_2.png caption=Example hysteretic relative permeability functions.  The system initialises at full saturation, dries to saturation 0.1 (less than $S_{l, r}$) and then wets back to saturation 1.  In the extended region ($S_{l} \leq S_{l, r}$), the drying curve equals the wetting curve.  id=histeretic_krel_example_2
+
+## Exploring hysteresis using the python script
+
+A python script that produces plots of hysteretic capillary pressure and relative permeability has been included in the MOOSE repository.  This allows users to quickly explore the consequences of choosing different extension strategies to the capillary and relative-permeability curves, as well as the impact of hysteresis order on the shape of the curves.  It was used to produce all the figures displayed on this page.  It is
+
+!listing modules/porous_flow/test/tests/hysteresis/hys.py
+
+
+## Input file syntax for hysteretic capillarity
+
+To include hysteretic capillarity in an existing (non-hysteretic) input file, the following changes need to be made.
+
+1. Any capillary-pressure UserObjects, such as [PorousFlowCapillaryPressureVG](PorousFlowCapillaryPressureVG.md), may be removed from the input file
+2. A [PorousFlowHysteresisOrder](PorousFlowHysteresisOrder.md) Material needs to be included.
+3. A Material that computes the porepressure(s) and saturation(s) needs to be included.
+
+   - For 1-phase partially-saturated situations, [PorousFlow1PhaseP](PorousFlow1PhaseP.md) should be removed and replaced by [PorousFlow1PhaseHysP](PorousFlow1PhaseHysP.md).  Note the van-Genuchten parameter input is $m$ for the non-hysteretic version, but $n$ for the hysteretic version, where $n = 1/(1 - m)$ and $m = 1 - 1/n$.
```

Comment:
No hyphen in van Genuchten

Reply:
oh my goodness, there were many instances of the hyphen in van-Genuchten!  Done